### ISSUE TITLE
Values for labels in the navigation menus and dropdowns of the website (top black bar)

### ISSUE THREAD

**Issue by eu-genia** [BetaMasaheft/Documentation#1573]
I think it might be helpful if the users, on clicking the top menu tabs, get a definition of the category they have landed in.
For example, I think the word "Clavis" is not understandbale to those external to the project. And then the points underneath are not all Clavis. Why not call the top menu point Works, and then have Clavis Aethiopica instead of Textual Units, and the definition of the Clavis Aethiopica on top of the https://betamasaheft.eu/works/list page.

Also for other categories (Manuscripts, Persons, Places), however self explicable they may seem, a short intro text may be helpful.

Can we build that in?

We can take as the starting point even the text which is now "hidden" under Help (https://betamasaheft.eu/help.html), maybe enhancing it by what Dorothea wrote for the Linking Manuscripts book, or similar.

What do you all think?

@PietroLiuzzo @thea-m @DenisNosnitsin1970 @DariaElagina

**Comment by thea-m** [comment]
I agree with anything which makes the app more approachable. Zhenia's suggestion seems sensible to me, but this particular point was already raised repeatedly (https://github.com/BetaMasaheft/Documentation/issues/1117, https://github.com/BetaMasaheft/Documentation/issues/1247), there might have been reasons for keeping "Clavis" in this prominent place which I forgot.

**Comment by eu-genia** [comment]
The issues above were a bit different, the result of the 1247 was the help page, but it is again not visible. Apart from definitions etc. I was wondering whether there is a problem with adding intro text I am not aware of.
It can also be a hovering box with definitions, I believe there was once a discussion on that as well, but just as it is now, noone apart from us knows where from the starting page they should go to find the works, and noone knows what Clavis is...

**Comment by PietroLiuzzo** [comment]
I abstain from comments and will behave as good slave once you have decided what to do. please assign me only once a final decision with a set of defined requirements has been made on this point.

**Comment by eu-genia** [comment]
My suggestion is:

- [ ] rename upper menu point Works

- [ ] rename Textual units Clavis Aethiopica

- [ ] introduce definitions for each subcategory on top of simple lists - here the question is whether there can be an intro page when clicking simply on the top tab word (Works / Manuscripts / Places) before having to select a submenu


We should agree on the text to appear. I can create suggestions that can be group edited here and then Pietro shall put them online.

**Comment by eu-genia** [comment]
Works

Clavis Aethiopica (CAe)
The project Beta maṣāḥǝft is working towards creating an exhaustive repertory of all works circulating in Ethiopian and Eritrean manuscript tradition. Every clearly identifiable textual unit receives a unique number, which scholars now may use to univocally refer to a specific text in their publications.
In the filter search offered here one can search for a work by its label, a keyword, but also directly by its CAe identifier - or, wherever known and provided, identifier used by other claves, including Bibliotheca Hagiographica Graeca (BHG), Corpus Patrum Graecorum (CPG), Clavis Coptica (CC), Clavis Apocryphorum Veteris Testamenti (CAVT), Clavis Apocryphorum Novi Testamenti (CANT), etc.

Narrative Units
The project additionally identifies Narrative Units to refer to recurring motifs or text types, where no clavis identification is possible or necessary. Frequently documentary additiones are assigned a Narrative Unit ID, or thematically clearly demarkated passages from various recensions of a larger work.

Documents corpora
This particular category is the result of cooperation between the project Beta maṣāḥǝft and the project Ethiopian Manuscript Archives. The EMA project, initially developed in 2010 by Anaïs Wion, and now part of the
larter project EthioChrisProcess - Christianization and religious interactions in Ethiopia (6th-13th century) : comparative approaches with Nubia and Egypt (ANR, 2018-2022, https://anr.fr/Project-ANR-17-CE27-0020), aims to edit and equip the corpus of administrative acts
of the Christian kingdom of Ethiopia, for medieval and modern periods. The list view shows the documentary collections encoded. 



((we probably should also add https://anr.fr/Project-ANR-17-CE27-0020 among the Partners))

**Comment by eu-genia** [comment]
Manuscripts
Producing online descriptions of (predominantly) Christian manuscripts from Ethiopia and Eritrea is the main scope of the Beta maṣāḥǝft project. We (1) gradually encode descriptions from printed catalogues, beginning from the historical ones, (2) incorporate digital descriptions produced by other projects, adjusting them wherever possible, and (3) produce descriptions of previously unknown and/or uncatalogued manuscripts. The encoding follows the TEI XML standards (for the encoding specifics you can check our guidelines at https://betamasaheft.eu/Guidelines/?id=manuscripts).


Shelf marks (full list)  ((NB: I would put this one first, as this is a complete unfiltered list not requiring search)

Here you can browse a full list of manuscripts available on the platform, arranged by repositories and shelf marks (clicking on the "show list" button will expand the list for each location).

Manuscripts (search)

In this search form you can search for (filter) all manuscripts encoded by the project. On the left side you get filters based on the indexes for that type of resources in the database, on the right side you will see, after searching, your results as in a table, paginated by 20 as per defaults. You can change the pagination option once you have done your search by entering a value in the "how many per page?" field of the pagination bar. Clicking on the hints button above will produce some additional blinking buttons which will provide additional guidance on the content.
For more guidance in lists and filters visit https://betamasaheft.eu/help.html.

Manuscript Images
The Mirador viewer can show all images of manuscripts we are currently serving or we have a usable link to a manifest for. You can either directly view the images in this mirador instance or click on the info box and go to the record for that manuscript.

Catalogues 
The list of catalogue sources used for our manuscript descriptions. Clicking on one of the titles will open a list view with all the manuscripts in that catalogue for which we have a record.
((Note here in the list there is a link at the bottom which is wrong))

Inscriptions
While the Beta maṣāḥǝft project focuses on manuscripts, inscriptions are often an inseparable part of the manuscript tradition and its direct precursors, therefore we also offer the encoding of the known inscriptions from Ethiopia and Eritrea wherever possible. Part of the encoding is carried out in Hamburg, part is the result of cooperation with other projects, such as DASI: Digital Archive for the Study of pre-islamic Arabian Inscriptions (http://dasi.cnr.it/). 
For more guidance in lists and filters visit https://betamasaheft.eu/help.html.



((We probably should also add Digital Archive for the Study of pre-islamic Arabian Inscriptions (http://dasi.cnr.it/) among the Partners))

((I remind that as far as I can see there is no menu tab or item bringing to https://betamasaheft.eu/availableImages.html, it should be added either under Manuscripts or either Resources))

**Comment by eu-genia** [comment]
Places
We create metadata for all places associated with the manuscript production and circulation as well as those mentioned in the texts used by the project. The encoding of places in Beta maṣāḥǝft will thus result in a Gazetteer of the Ethiopian tradition. We follow the principles established by Pleiades (https://pleiades.stoa.org/places) and lined out in the Syriaca.org TEI Manual and Schema for Historical Geography (http://syriaca.org/geo/index.html) which allow us to distinguish between places, locations, and names of places.
Place records should ideally contain the attested names of the place in local languages and translation, including possible variants, as well as any information available on the foundation of the place, its existence and development. Coordinates can be added or will be retrieved if a reference to the place’s Wikidata ID is given.
As this is a work in progress, and many records were inherited from the Encyclopaedia Aethiopica, there are still many inconsistencies that we are trying to gradually fix.

Places
This tab offers a filterable list of all available places. For more guidance in lists and filters visit https://betamasaheft.eu/help.html.

Repositories
Repositories are those locations where manuscripts encoded by the project are or used to be preserved. While they are encoded in the same way as all places are, the view offered is different, showing a list of manuscripts associated with the place. For more guidance in lists and filters visit https://betamasaheft.eu/help.html.

**Comment by eu-genia** [comment]
Persons
We create metadata for all persons associated with the manuscript production and circulation (rulers, religious authorities, scribes, donors, and commissioners) as well as those mentioned in the texts used by the project. The encoding of persons in Beta maṣāḥǝft will thus result in a comprehensive Prosopography of the Ethiopian tradition. Records shoulod contain the person’s original and transliterated names and basic information on their life and occupations as well as a reference to their Wikidata ID, if existing.
As this is a work in progress, and many records were inherited from the Encyclopaedia Aethiopica, there are still many inconsistencies that we are trying to gradually fix.
For more guidance in lists and filters visit https://betamasaheft.eu/help.html.

**Comment by eu-genia** [comment]
@abausi please also have a say whether the suggested intro definitions above are helpful/needed/to be improved

**Comment by abausi** [comment]
I find all this excellent and I have no objeection. I would only insist that, as in fact it was suggested, "Clavis" should remain, even if in a less prominent place. To be honest: the fact that the concept and word of "Clavis" is not understood, reveals a problem, and it is not _the_ problem. The problem is that Ethiopian and Eritrean studies are not so connected (as in fact they should be, given the huge common cultural background with biblical studies and patristic studies) with other fields, where everyone knows what CPG, CANT etc. are. So, let's be aware of this; let's have an as generally accessible website as possible; but do not let's forget that the real problem is the scarce familiarity with the other "Claves". For sure, to introduce and present in a more accessible way the "CAe" will also have the desirable effect that also the other "Claves" will become more familiar and accessible.

**Comment by DariaElagina** [comment]
I think that it is quite confusing to have Works as a general category in the menu, and then again as a subcategory. Maybe we can use the word Texts? Works and Narrative Units are at the end texts, I think.

**Comment by eu-genia** [comment]
There is no subcategory "Works" - I was referring to the top level tab. It can be "Texts" as well, OK for me.
The 3 subcategoris are
- Clavis aethiopica
- Narrative units
- Documentary corpora

**Comment by abausi** [comment]
In order to have a better idea, we should have a sort of stemma, or it becomes difficult to understand exactly the hierarchy of the proposal. But the definitions seem excellent.

**Comment by eu-genia** [comment]
I refer to the top menu as it is and the submenus, the stemma is the one already online

**Comment by eu-genia** [comment]
![image](https://user-images.githubusercontent.com/17987728/100105483-b77f1400-2e67-11eb-925b-8fa440e83d9a.png)

**Comment by DariaElagina** [comment]
> 
> 
> There is no subcategory "Works" - I was referring to the top level tab. It can be "Texts" as well, OK for me.
> The 3 subcategoris are
> 
>     * Clavis aethiopica
> 
>     * Narrative units
> 
>     * Documentary corpora

Sure! But in this case Works (in our definition) refer only to Clavis, neither to Narrative Units, nor to Documents. And then one could also put the definition of Clavis and Works together, because our Clavis is a repertory of Works. That's why I think that we should pick up a broader category for the tab.
Texts:
Clavis (Works)
Narrative units
Documentary corpora

**Comment by eu-genia** [comment]
OK for me will edit the text above

**Comment by PietroLiuzzo** [comment]
https://github.com/BetaMasaheft/guidelines/issues/106

**Comment by PietroLiuzzo** [comment]
@SusanneHummel @SophiaD-M @Ralph-Lee-UK @MarcinKrawczuk @antobrita @smaugustine

**Comment by DenisNosnitsin1970** [comment]
"Clavis (Works)
Narrative units
Documentary corpora"
is fine.

**Comment by antobrita** [comment]
The problem I see with having "Texts" at the top level tab, is that it excludes illuminations that could be included in Narrative Units (e.g. a specific miracle or any other episode can have a literary form - text -, a visual form - miniature -, or even both). The categories Narrative Unit and Textual Unit are conceptually different: the first is related to the content and the second to the codicological shape of this content.

I would then propose:

**Works** [top level tab; by Works I mean Manuscript Content]
**Texts (Clavis Aethiopica)**  or  **Literary texts (Clavis Aethiopica)** [in order to maintain "Clavis Aethiopica"]
**Documents Corpora** or **Archival Texts** or **Archival Documents**
**Textual Units**
**Narrative Units**
[One could even think to add **Illuminations** to the list, but maybe you already discussed this issue]

I would also invert in the top level tab (1) **Manuscripts** and (2) **Works**, so that the category **Works**, between **Manuscripts** and **Places** can be more easily understood

**Comment by eu-genia** [comment]
I am not sure I understand what @antobrita means under Illuminations, but if Art themes are meant, they are treated with Keywords here https://betamasaheft.eu/authority-files/list (Art Themes - there are all the Miracles for example) - or here https://betamasaheft.eu/decorations. We might also promote the Keyword list (now under Indexes) to an upper level or add more links from Works (to the Art Themes index, or also to the Decorations, or also to Additions and Titles indices, might be helpful)
OK to the change of sequence, we start from the manuscript

**Comment by eu-genia** [comment]
> The categories Narrative Unit and Textual Unit are conceptually different: the first is related to the content and the second to the codicological shape of this content.

I am also not sure what is meant here. This has already been addressed here https://github.com/BetaMasaheft/guidelines/issues/106 and here https://betamasaheft.eu/Guidelines/?id=definitionWorks
For us, textual unit = work = clavis ID. Narrative unit is another level, but also related to content.

**Comment by antobrita** [comment]
> I am not sure I understand what @antobrita means under Illuminations, but if Art themes are meant, they are treated with Keywords here https://betamasaheft.eu/authority-files/list (Art Themes - there are all the Miracles for example) - or here https://betamasaheft.eu/decorations. We might also promote the Keyword list (now under Indexes) to an upper level or add more links from Works (to the Art Themes index, or also to the Decorations, or also to Additions and Titles indices, might be helpful)
> OK to the change of sequence, we start from the manuscript

Yes, Zhenia. This is what I meant (sorry, I'm still learning how to swim into BM).
As far as I understand, in the item of the top level tab that until now was called **Clavis** you wish to include the content of the manuscripts. If this is the case, I think that at least Art Themes shall be included there since they are an important component of the manuscript content (from my experience in the CSMC I know that art historians are very sensitive to this topic and often felt discriminated when some of us called "text" the content of manuscripts). But if this is not the case, and you wish to include in the old item **Clavis** only texts, then I think that replacing **Clavis** with **Texts** does make a lot of sense.

Another solution could be to add in the top level tab **Art Themes** besides **Manuscripts** and **Texts**. At the moment it is very difficult to find Art Themes in BM if you do not know where to look for it (it is hidden under Indexes and then under Key Words). And people who are not familiar with BM won't  find it easily.

**Comment by antobrita** [comment]
For us, textual unit = work = clavis ID. Narrative unit is another level, but also related to content.

I see how you understand these concepts, but I think that this way is a bit problematic for the Ethiopic ms tradition (the topic in itself is problematic, la verità). As far as I understand, your concept is based on Orlandi's work, which is a wonderful work, but based on  the Coptic tradition which is very much peculiar (dismembered manuscripts and fragments are the rule) and rarely finds parallels in the Ethiopic ms tradition.

As far as I understand these concepts (in relation to Ethiopic mss), a  Work (=Clavis ID) is text-related concept while a Texual unit is manuscript-related concept (that is the layout arrangement of a Work in the manuscript). A Work (=Clavid Id) can be written in one single Textual unit in a manuscrits (eventually divided in sub-textual units) but not necessarily so. On the other hand, two different Works (=Clavis Id) can be written in one manuscript as two different Textual units and in another manuscript as one single textual unit.

**Comment by antobrita** [comment]
> For us, textual unit = work = clavis ID. Narrative unit is another level, but also related to content.

I see how you understand these concepts, but I think that this way is a bit problematic for the Ethiopic ms tradition (the topic in itself is problematic, la verità). As far as I understand, your concept is based on Orlandi's work, which is a wonderful work, but based on the Coptic tradition which is very much peculiar (dismembered manuscripts and fragments are the rule) and rarely finds parallels in the Ethiopic ms tradition.

As far as I understand these concepts (in relation to Ethiopic mss), a Work (=Clavis ID) is text-related concept while a Texual unit is manuscript-related concept (that is the layout arrangement of a Work in the manuscript). A Work (=Clavid Id) can be written in one single Textual unit in a manuscrits (eventually divided in sub-textual units) but not necessarily so. On the other hand, two different Works (=Clavis Id) can be written in one manuscript as two different Textual units and in another manuscript as one single textual unit.

**Comment by eu-genia** [comment]
> As far as I understand, in the item of the top level tab that until now was called **Clavis** you wish to include the content of the manuscripts. If this is the case, I think that at least Art Themes shall be included there since they are an important component of the manuscript content (from my experience in the CSMC I know that art historians are very sensitive to this topic and often felt discriminated when some of us called "text" the content of manuscripts). But if this is not the case, and you wish to include in the old item **Clavis** only texts, then I think that replacing **Clavis** with **Texts** does make a lot of sense.

Yes, the tab should relate to textual content, as otherwise too many other things would be included here. Only texts are to be looked for here.

> Another solution could be to add in the top level tab **Art Themes** besides **Manuscripts** and **Texts**. At the moment it is very difficult to find Art Themes in BM if you do not know where to look for it (it is hidden under Indexes and then under Key Words). And people who are not familiar with BM won't find it easily.

This I would tend to agree with.

>I see how you understand these concepts, but I think that this way is a bit problematic for the Ethiopic ms tradition (the topic in itself is problematic, la verità). As far as I understand, your concept is based on Orlandi's work, which is a wonderful work, but based on the Coptic tradition which is very much peculiar (dismembered manuscripts and fragments are the rule) and rarely finds parallels in the Ethiopic ms tradition.
>As far as I understand these concepts (in relation to Ethiopic mss), a Work (=Clavis ID) is text-related concept while a Texual unit is manuscript-related concept (that is the layout arrangement of a Work in the manuscript). A Work (=Clavid Id) can be written in one single Textual unit in a manuscrits (eventually divided in sub-textual units) but not necessarily so. On the other hand, two different Works (=Clavis Id) can be written in one manuscript as two different Textual units and in another manuscript as one single textual unit.

Well, one can invent all sort of taxonomies and definitions and the terms we all agree are vague. 
From the onset, we did take Orlandi's work as the basis for our definitions, in order to have a point of reference and avoid confusion as much as possible - and for our Clavis we are only interested in **works**. 
If this is so confusing we can stop talking about "textual units" in this sense (and I did not use it in the intro proposed above). As for textual units understood as **units of layout** - this is an all different layer, which is not relevant for our definition of a **work**.

**Comment by abausi** [comment]
In fact, we started from Orlandi because his contribution invited us to look at the evidence and not to (especially interlingual) partial correspondences. Orlandi would be the first one not to propose himself as an untouchable authority. It is true what Antonella says, that the Coptic tradition is predominantly a single-witness tradition (in philological terms), whereas Ethiopic single-witness traditions are the exceptions, with all imaginable consequences.

In order to go on with the discussion, I would invite each one to propose synthetically the alternative hierarchy proposed, e.g.:
1 = XXXX
1.1 = XXXX
1.2 = XXXX
1.3 = XXXX
2 = XXXX
etc.

It will be much easier to understand what is actually proposed.

**Comment by eu-genia** [comment]
New tab:
Art themes
While encoding the information on the decorations present in the manuscripts, the project Beta maṣāḥǝft aims at creating an exhaustive repertory of art themes and techniques present in Ethiopian and Eritrean Christian tradition. You can check our encoding guidelines at https://betamasaheft.eu/Guidelines/?q=art%20theme&id=decorationDescription.
Two types of searches for aspects of manuscript decoration are possible, the decorations filtered search and the general keyword search.

Submenu 1: Index of decorations (https://betamasaheft.eu/decorations)
The decorations filtered search, originally designed with Jacopo Gnisci, looks at decorations and their features only. The filters on the left are relative only to the selected features, reading the legends will help you to figure out what you can filter. For example you can search for all encoded decorations of a specific art theme, or search the encoded legends. If the decorations are present, but not encoded, you will not get them in the results. If an image is available, you will also find a thumbnail linking to the image viewer for that manuscript.

Submenu 2: Keywords list (https://betamasaheft.eu/authority-files/list)
You can search for particular motifs or aspects, including style, also through the keyword search. Just click on "Art keywords" and "Art themes" on the left to browse through the options.

**Comment by eu-genia** [comment]
1. Manuscripts
1.1. Shelf marks 
1.2. Manuscripts search
1.3. Image view
1.4. Catalogues encoded
1.5. Inscriptions

2. Texts
2.1. Clavis Aethiopica (CAe)
2.2. Narrative units
2.3. Documents corpora

3. Art themes
3.1. Index of decorations
3.2. Art keywords

4. Places
4.1. Place names
4.2. Repositories

5. Persons
5.1. Persons and groups
5.2. Ethnic units

6. Indexes
as now

7. Resources

8. Projects

**Comment by antobrita** [comment]
> In order to go on with the discussion, I would invite each one to propose synthetically the alternative hierarchy proposed, e.g.:
> 1 = XXXX
> 1.1 = XXXX
> 1.2 = XXXX
> 1.3 = XXXX
> 2 = XXXX
> etc.

I would propose:
1 = **Texts**
1.1 = **Literary Texts (Clavis Aethiopica)**
1.2 = **Narrative Units**
1.3 = **Archival Texts (Documentary Corpora)**

and, as Zhenia already suggested:

2. = **Art Themes**
2.1 = **Index of decorations**
2.1 = **Keywords list**

**Comment by eu-genia** [comment]
@antobrita, did not you suggest above to have first 1. Manuscripts and 2. Texts?

**Comment by antobrita** [comment]
> @antobrita, did not you suggest above to have first 1. Manuscripts and 2. Texts?

yes, Zhenia. Sorry, I thought in the above message we were referring only to the two items Texts and Art themes and that the order 1 and 2 did not reflect the sequence in which they appear

**Comment by antobrita** [comment]
> 1. Manuscripts
>    1.1. Shelf marks
>    1.2. Manuscripts search
>    1.3. Image view
>    1.4. Catalogues encoded
>    1.5. Inscriptions
> 2. Texts
>    2.1. Clavis Aethiopica (CAe)
>    2.2. Narrative units
>    2.3. Documents corpora
> 3. Art themes
>    3.1. Index of decorations
>    3.2. Art keywords
> 4. Places
>    4.1. Place names
>    4.2. Repositories
> 5. Persons
>    5.1. Persons and groups
>    5.2. Ethnic units
> 6. Indexes
>    as now
> 7. Resources
> 8. Projects

I saw only now your proposal, I agree with that. Please ignore my proposal, I will delete the message, to avoid confusion

**Comment by Ralph-Lee-UK** [comment]
This all seems to make very good sense to me, and the detailed points raised as the discussion develops seem well addressed. 

My only thought goes to the broad classes that we created for various purposes. One was a broad 'asmat prayer' category which is LIT5888AsmatPrayer. This was not created as a specific work, but in order to be consistent with situations where asmat prayers are identified as additions with a type="Asmat" so that for future research all instances of such prayers may be identified. 

Along similar lines we defined several 'Amharic commentary on....' work files to identify broadly commentary material on a specific work, where at the moment not enough research has been done to identify different versions of such commentaries. 

Both of these cases are not specific works, but are important records, and perhaps in the menu there should be a category that identifies and explains these?

**Comment by abausi** [comment]
To sum up, and close this issue: let's stay with the last proposal by @eu-genia (whereupon, if I see correctly, also @antobrita agrees), but keeping on considering under Clavis Aethiopica the very important category recalled by @Ralph-Lee-UK that is essential and has not only a practical justification. If I remember well, this was related to the category of "general", that was discussed in another issue.

**Comment by eu-genia** [comment]
@Ralph-Lee-UK makes a valid comment, which is more relevant for those who edit (= guidelines) than for external users, but is still important for understanding of the Clavis. I have now added a sentence to the definition above that addresses the problem.

**Comment by SophiaD-M** [comment]
If I may get involved? 
I do not understand the link below:
https://betamasaheft.eu/manuscripts/browse

It is found under the header Manuscripts -> Shelf marks. But the list is not a list of shelfmarks (which are DD, or EMML, or similar), but it is a list of the repositories, am I right? 
The list is wonderful, very helpful and should not be removed!! I just do not understand why it is called "shelfmarks". If I search EMML mss in this link, I have to go to Hill Museum... thus to the repository. 

And a purely aesthetical-comment, the size of the words is really huge, could it be maybe smaller? 
Also to make scrolling easier could there be a list of the alphabet above? So I could just jump to Z if I wanted, instead of scrolling? And likewise a "go back up" button to save scrolling back up?

**Comment by smaugustine** [comment]
Just to point out, as it is the number and length of the labels is very long, I'm not sure that making it even longer is the best idea? See, for example, how it is now on the widest possible window size on my laptop:

<img width="1280" alt="Screen Shot 2020-11-26 at 14 34 51" src="https://user-images.githubusercontent.com/6315671/100357137-8d0e9180-2ff4-11eb-90cb-7d51bbf36456.png">

We could tweak the styling to fit more in, but I wonder that we should take a more economical approach to what labels run along the top and what are in the dropdowns.

**Comment by smaugustine** [comment]
Dropdowns that take advantage of horizontal space, for example, might be an option to consider. Example:

![Screen Shot 2020-11-26 at 14 37 50](https://user-images.githubusercontent.com/6315671/100357499-0d34f700-2ff5-11eb-86fd-37184fe8fe48.png)

**Comment by PietroLiuzzo** [comment]
I am afraid I will not go too much into design. The length of the labels is indeed an issue for a top bar. Some of them disappear on smaller screens and are grouped in a menu. 

- Image view ---> Images Viewer 
- Catalogues encoded ---> catalogues (is not true that they are encoded and the list is not a list of encoded catalogues!)
- Art Theme may be Art only (why do this now become a top level thing? we have very very little... plus, see below.)
- Index of decorations --> Decorations

The current list involves not only a change in the items but a re-organization of the related views. This is more problematic IMHO and risks to hide behind an apparent wish of clarification a series of  separate requirements for further features and lists, etc. which as such, i.e. introducing changes, will inevitably produce confusion, so going against the purpose of this effort. I would suggest caution. For example:
 
- we do not have a list of place names, only one of places, which by request, actually includes repositories although these are separate entities conceptually
- we do not have a separate list of Art keywords
- Ethnic and groups have been repeatedly suggested and discarded in favour of the full list

**Comment by smaugustine** [comment]
I think we should acknowledge that not *every* page needs to be linked from the top bar, only those for which a user needs quick access. Those that aren't can be linked to from other pages. Think about the end-users who will be overwhelmed by a jumble of links.

- I would suggest that "Projects," for example, does not need its own item, we could link to a page listing the projects from "Resources."
- A link to the image viewer, I think, is unneeded, since a user would first navigate to the manuscript record and then click "Images;" I don't think a link to an empty iiif viewer is helpful to most users.
- Unless I'm mistaken, art themes are only relevant in the context of manuscripts, and so it could be placed there.
- The Zotero library could be linked to from the bibliography page instead of having its own item

Merely as a rough suggestion, simply to outline the sort of economical approach I would encourage:

0. [Beta Masaheft Logo]
   1. Home
   2. About (combines "Team", "Partners", and "Contacts")
   3. Guidelines
   4. Data (combines "XPath" and "SPARQL")
   5. API
1. Manuscripts
   1. Shelf Marks
   2. Search
   3. Catalogues
   4. Inscriptions
   5. Art Themes
   6. Compare
   7. Map
2. Texts
   1. Clavis
   2. Narrative Units
   3. Related Units
3. Places
   1. Place Names
   2. Repositories
4. Persons
5. Resources
   1. Bibliography
   2. Indexes
   3. Projects
   4. Gender & Manuscripts
   5. Parser
   6. Lexicon
6. Help
7. 🔍

**Comment by smaugustine** [comment]
Regarding the possibility of doing something more complex, such as I suggested with a "horizontal dropdown," that would allow for more links and clearer hierarchies, I would be willing to take on implementing the design and functionality, if this is something deemed desirable.

**Comment by PietroLiuzzo** [comment]
I do not think we need to invest this time in development and design when there are so many manuscripts still needing encoding. So, as much as your offer for help is appreciated, I think it is not a good way to invest project time.
In the same way I do not think that re-discussing issues which have been discussed at length in the past is actually bringing something to the project. If more of these requirements come along as they are, on colors and disposition of information in the website, which we have always tried to leave to the minimum, I suggest that we rather and more organically collect these requests for a major update next year, but it will have to come with a few other things, which will follow the planned user based enquiry.

**Comment by eu-genia** [comment]
> * Art Themes

as @antobrita noted, art is rather an aspect of Content, alongside Texts. So if no space available for a separate tab, it should not be probably grouped under Manuscripts but together with Texts as e.g.
 
2. Content (then the intro text has to be changed of course)
- Clavis
- Narrative Units
- Documents
- Art Themes

**Comment by PietroLiuzzo** [comment]
from  https://github.com/BetaMasaheft/Documentation/issues/1597

Have a separate menu point (can stay under Resources - or top level, but again the width may be an issue) for Search, where all Searches have sub-menu-points
Menu: SEARCH
Submenu 1: Simple search
Submenu 2: Facet search
Submenu 3: Advanced search
Submenu 4: X-Path query
Submenu 5: SPARQL query

**Comment by MarcinKrawczuk** [comment]
I really don't feel qualified to take part in this discussion but since I was asked to contrubite... My first though was: does the BM website include any functions for people with disabilities? I've heard opinions that installing such functions (or even thinking about them) often leads to improvements for all users.

**Comment by PietroLiuzzo** [comment]
I am adding these values and reorganizing the navigation bar. What do you think about the following ? 
<img width="2033" alt="Bildschirmfoto 2021-09-08 um 13 59 31" src="https://user-images.githubusercontent.com/508012/132505563-0759cfca-7d02-4b69-a6dc-17b3539fbd21.png">
What will happen here (where I have not yet reorganized anything) is that the main menu description will stick on the right, the options (submenus) will be like you have seen before on the right spreading three quarters of the width and hovering on them the additional explanation will popup in red below the list.
@eu-genia @thea-m @DariaElagina @DenisNosnitsin1970

**Comment by abausi** [comment]
I do not understand what appears on the left: is everything on the right?

Il 08.09.2021 14:21, Pietro Liuzzo ha scritto:

**Comment by PietroLiuzzo** [comment]
I have made a screencast, hoping this becomes more clear.
https://www.awesomescreenshot.com/video/5159551?key=02010737a377bf010b780a0b20989267

**Comment by abausi** [comment]
Thank you, Pietro: I like it

Il 09.09.2021 09:04, Pietro Liuzzo ha scritto:

**Comment by PietroLiuzzo** [comment]
@eu-genia @thea-m @DariaElagina @DenisNosnitsin1970 please, if you also like it as solution, leave a sign, I will then close this issue.

**Comment by eu-genia** [comment]
looks good, thank you
